package.json

```diff
@@ -7,44 +7,40 @@
     "type": "git",
     "url": "https://github.com/OfficeDev/office-ui-fabric-react"
   },
+  "devDependencies": {
+    "beachball": "^1.9.2",
+    "git-hooks": "^1.1.10",
+    "lerna": "^3.15.0",
+    "lint-staged": "^7.0.5"
+  },
   "scripts": {
-    "_invalidateCache": "node ./scripts/invalidate-just-cache.js",
-    "_rushInstall": "node common/scripts/install-run-rush.js install --purge",
-    "_rushBuild": "node common/scripts/install-run-rush.js build",
-    "_rushBuildToFabricResources": "node common/scripts/install-run-rush.js build --to @uifabric/fabric-website-resources",
-    "_rushRebuild": "npm run _invalidateCache && node common/scripts/install-run-rush.js rebuild --production --verbose --lint",
-    "_rushRebuildCi": "npm run _invalidateCache && node common/scripts/install-run-rush.js rebuild --verbose --lint",
-    "_rushRebuildFast": "npm run _invalidateCache && node common/scripts/install-run-rush.js rebuild",
-    "postinstall": "npm run _rushInstall && npm run _rushBuildToFabricResources -- --npm-install-mode",
-    "test": "npm run _rushBuild",
+    "preinstall": "node ./scripts/use-yarn-please.js",
+    "postinstall": "node ./scripts/monorepo/buildTo.js @uifabric/fabric-website-resources --min",
```

Comment:
Will this cause a new build every time someone does `yarn install`? This made sense with npm as `npm install` was rarely used, but this feels like it might be excessive.

Reply:
`npm install` was actually used all the time ... and indeed it was painful. People had to resort to crack open the tooling underneath to figure out how to do this. If @dzearing is okay, I'd love to REMOVE this postinstall script altogether and train people to do this:

```
yarn
yarn buildto @uifabric/fabric-website-resources (maybe i can create a shorter thing there)
yarn start
```

Reply:
> `yarn buildto @uifabric/fabric-website-resources (maybe i can create a shorter thing there)`

Yes please...that is WAY too much to ask people to type/remember.

Reply:
Also, the ability to use short project names (fabric-website-resources), which Rush supported, would be super helpful. But even with shorter name support you should definitely still make a shortcut command.